vagrant/Vagrantfile

```diff
@@ -71,6 +71,7 @@ Vagrant.configure("2") do |config|
     node.vm.box = "ubuntu/xenial64"
     node.vm.hostname = "frontend"
     node.vm.network :private_network, ip: FRONTEND_IP
+    node.vm.network "forwarded_port", guest: 8080, host: 8080, guest_ip: "127.0.0.1"
```

Comment:
Let's remove it and use `vagrant ssh`-based solution instead (see previous comment)

Reply:
moved this step to the readme, mtls and trafficpermissions still works